index.css

```diff
@@ -0,0 +1,884 @@
+  color: #f72210;
+}
+
+.subscribe {
+  padding-top: 30px;
+  padding-bottom: 40px;
+  margin-right: 10px;
+  background-color: #f4f5f6;
+}
+
+#subscribeImgs {
+  position: relative;
+  margin-left: 30px;
+  width: 30%;
+}
+
+.overlap {
+  position: absolute;
+  top: 16px;
+  left: 80px;
+}
+
+#subscribeInfo {
+  width: 65%;
+  padding: 20px 40px;
+}
+
+#subscribeInfo h3, #newsletterInfo h3 {
+  font-size: 24px;
+  line-height: 28px;
+  font-weight: bold;
+}
+
+#subscribeInfo p, #newsletterInfo p {
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 40px;
+  font: 300 15px/21px "Helvetica Neue",Helvetica,Arial,sans-serif;
+}
+
+#subscribeButton, #signUpButton {
+  padding: 10px 10px 10px 12px;
+  text-decoration: none;
+  background-color: #f72210;
+  color: white;
+  font: bold 18px/18px "Roboto Condensed",sans-serif;
+}
+
+#subscribeButton:hover {
+  background-color: #de1f0e;
+}
+
+#newsletterImg {
+  width: 50%;
+  height: 296px;
+  background: url(https://g.newsweek.com/themes/newsweek/images/subscribe-img2.gif) center center / cover no-repeat;
+}
+
+#newsletterInfo {
+  padding: 30px 20px;
+  width: 50%;
+}
+
+footer {
+  border-top: 1px solid #ddd;
+  padding: 30px;
+}
+
+#footerWrapper {
+}
+
+.footerLeft {
+  width: 50%;
+  padding-left: 30px;
+}
+
+.footerLeft svg {
+  width: 236px;
+  height: 36px;
+  margin-right: 15px;
+  margin-bottom: 12px;
+  fill: #f72210;
+}
+
+.copyright {
+  font-size: 12px;
+  line-height: 12px;
+  color: #666;
+  margin-bottom: 12px;
+  font-family: Arial,Helvetica,sans-serif;
+}
+
+.footerRight {
+  width: 50%;
+  padding-right: 30px;
+}
+
+.footerRight i {
+  color: #666;
+  font-size: 22px;
+  margin: 10px;
+}
+
+.footerRight i:hover {
+  color: #3b5998;
+}
+
+.footerBottom {
+  border-top: 1px solid #ddd;
+}
+
+.footerBottom li {
+  color: #666;
+}
+
+.footerBottom a {
+  padding: 0 12px;
+  border-right: 1px solid #ddd;
+  font-size: 15px;
+  color: #666;
+}
+
+.list-left {
+  width: 60%;
+  padding: 10px 40px;
+}
+
+.list-right {
+  width: 40%;
+  padding: 10px 60px;
+}
+
+.list-right a {
+  padding: 0 10px;
+  line-height: 1;
+}
+
+.stickyBar {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: #f72210;
+  height: 45px;
+  width: 100%;
+  z-index: 1;
+}
+
+.stickyWrapper {
+  width: 100%;
+  border: solid blue 1px;
+}
+
+.stickyLeft svg {
+  width: 132px;
+  height: 20px;
+  fill: white;
+}
+
+.stickyLeft {
+  padding-left: 15px;
+  width: 50%;
+}
+
+.stickyRight {
+  padding-right: 15px;
+  width: 50%;
+}
+
+.stickyRight a {
+  text-decoration: none;
+  padding: 6px 12px;
+  font: bold 18px/14px "Roboto Condensed",sans-serif;
+  height: max-content;
+}
+
+#stickySign {
+    color: white;
+}
+
+#stickySubs {
+  background: white;
+  color: red;
+  margin-right: 15px;
+}
+
+.stickyRight i {
+  color: #fb9a94;
+  font-size: 30px;
+  width: max-content;
+}
+
+
+@media (max-width: 992px) {
+  #wrapper {
+    width: 100%;
+  }
+
```

Comment:
- Again these media queries are necessary. You supposed to use bootstrap classes.